### ISSUE TITLE
Cannot fs.unlink() after getXlsxStream().on('end')

### ISSUE THREAD

**Issue by kubo0212monsterpixel** [Claviz/xlstream#13]
The original file does not remove until stop nodejs run. I think zip() is not close stream after finish stream data.

**Comment by jansivans** [comment]
Can you provide a code to reproduce this issue please?

**Comment by kubo0212monsterpixel** [comment]
```
const data = [];
const stream = await getXlsxStream({
    filePath: `data_${index}_${fileCreateAt}.xlsx`,
    sheet: 0,
    withHeader: true,
    ignoreEmpty: true,
});

stream.on('data', value => {
    data.push(value);
};

stream.on('end', () => {
    try {
        fs.unlinkSync(`data_${index}_${fileCreateAt}.xlsx`);
    } catch (err) {
        // no error here
    }
}
```

**Comment by kubo0212monsterpixel** [comment]
I tried `stream.destroy()` but it still does not lose the original file until I stop nodejs process.

**Comment by jansivans** [comment]
Can you provide a .xlsx file? Can't reproduce with mine.

**Comment by kubo0212monsterpixel** [comment]
Anything file `.xlsx` even empty file.

**Comment by jansivans** [comment]
As you can see on this GIF file gets deleted without any exception.
![no-error](https://user-images.githubusercontent.com/5667073/77892396-3ee59a80-727b-11ea-94ad-dc4e5d1add8f.gif)

**Comment by kubo0212monsterpixel** [comment]
Before stop nodejs
![be4](https://user-images.githubusercontent.com/34226751/77893299-f2f81d00-729d-11ea-89b6-73135c6e9625.png)

After stop nodejs
![after](https://user-images.githubusercontent.com/34226751/77893304-f4294a00-729d-11ea-8a29-fa370de45b5d.png)

**Comment by jansivans** [comment]
Ok, I understood. Will be fixed ASAP.

**Comment by jansivans** [comment]
Version 1.1.2 was published to npm. Please check if issue is fixed.

**Comment by kubo0212monsterpixel** [comment]
Its resolved at v1.1.2 (old version is 1.1.1). Thank you all for your quick and helpful.

**Comment by jansivans** [comment]
Thanks for reporting.
